src/mbuf/mbuf.c

```diff
@@ -544,6 +544,27 @@ int mbuf_printf(struct mbuf *mb, const char *fmt, ...)
 }
 
 
+/**
+ * Print a safe formatted string to a memory buffer
+ *
+ * @param mb  Memory buffer
+ * @param fmt Formatted string
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int _mbuf_printf_s(struct mbuf *mb, const char *fmt, ...)
```

Comment:
perhaps the mbuf changes can be added as a phase 2 ?

Reply:
Ok will exclude this commit.